src/preload.ts

```diff
@@ -0,0 +1,53 @@
+/* eslint-env browser */
```

Comment:
We should set the `env` in xo instead: https://github.com/sindresorhus/caprine/blob/master/package.json#L64-L67

Reply:
I didn't want to do this since this is the only file that will have the browser environment.  Unless the others should?

Reply:
I think it's alright to have globally set.